eclair-core/src/test/scala/fr/acinq/eclair/payment/Bolt11InvoiceSpec.scala

```diff
@@ -41,6 +41,46 @@ class Bolt11InvoiceSpec extends AnyFunSuite {
   val nodeId = pub
   assert(nodeId == PublicKey(hex"03e7156ae33b0a208d0744199163177e909e80176e55d97a2f221ede0f934dd9ad"))
 
+  // Copy of Bolt11Invoice.apply that doesn't strip unknown features
+  def createInvoiceUnsafe(chainHash: ByteVector32,
+            amount: Option[MilliSatoshi],
+            paymentHash: ByteVector32,
+            privateKey: PrivateKey,
+            description: Either[String, ByteVector32],
+            minFinalCltvExpiryDelta: CltvExpiryDelta,
+            fallbackAddress: Option[String] = None,
+            expirySeconds: Option[Long] = None,
+            extraHops: List[List[ExtraHop]] = Nil,
+            timestamp: TimestampSecond = TimestampSecond.now(),
+            paymentSecret: ByteVector32 = randomBytes32(),
+            paymentMetadata: Option[ByteVector] = None,
+            features: Features[InvoiceFeature] = defaultFeatures): Bolt11Invoice = {
```

Comment:
This could be even better and simply take `Features[FeatureScope]`, which lets us easily test known, non-invoice features in invoices?